app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -785,4 +777,13 @@ class MapActivity : ComponentActivity() {
             }
         }
     }
+
+    companion object {
+        private val INITIAL_POSITION = GeoPoint(46.518726, 6.566613)
+        private const val INITIAL_ZOOM = 18.0
+        private const val MARKER_SIDE_LENGTH = 100
+        private const val MAX_CLOSE_LOCATION_DISTANCE = 10.0    // meters, used to determine
+                                                                // if the player is close enough to
+                                                                //a location to interact with it
```

Comment:
An entire one line comment would be better than this 3 lines one that tries to match the 80 max char per line

Reply:
As you so wish